static/src/javascripts/lib/detect.js

```diff
@@ -219,6 +219,9 @@ const hasCrossedBreakpoint = (includeTweakpoint: boolean): Function => {
     };
 };
 
+// https://developer.apple.com/documentation/apple_pay_on_the_web/apple_pay_js_api/checking_for_apple_pay_availability
+const applePayApiAvailable = window.ApplePaySession;
```

Comment:
It's probably worth noting this isn't a boolean so `(applePayApiAvailable  === true)` would never return true.

It might be worth being a more explicit:
`const applePayApiAvailable: boolean = !!window.ApplePaySession;`